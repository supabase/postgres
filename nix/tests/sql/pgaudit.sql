-- Note: there is no test that the logs were correctly output. Only checking for exceptions
set pgaudit.log = 'write, ddl';
set pgaudit.log_relation = on;
set pgaudit.log_level = notice;

create schema v;

create table v.account(
  id int,
  name text,
  password text,
  description text
);

insert into v.account (id, name, password, description)
values (1, 'user1', 'HASH1', 'blah, blah');

select
  *
from
  v.account;

drop schema v cascade;
