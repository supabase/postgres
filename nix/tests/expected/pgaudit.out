-- Note: there is no test that the logs were correctly output. Only checking for exceptions
set pgaudit.log = 'write, ddl';
set pgaudit.log_relation = on;
set pgaudit.log_level = notice;
create schema v;
create table v.account(
  id int,
  name text,
  password text,
  description text
);
insert into v.account (id, name, password, description)
values (1, 'user1', 'HASH1', 'blah, blah');
select
  *
from
  v.account;
 id | name  | password | description 
----+-------+----------+-------------
  1 | user1 | HASH1    | blah, blah
(1 row)

drop schema v cascade;
NOTICE:  drop cascades to table v.account
